libs/design/article/src/article/article.component.scss

```diff
@@ -100,18 +100,23 @@
 
 	pre {
 		border-radius: 0.25rem;
-		font-size: t.$font-size-sm;
-		line-height: 1.5rem;
-		margin: 1.5rem 0;
-		padding: 1.5rem;
-		overflow-x: auto;
+		overflow: hidden;
+		position: relative;
 
 		&:last-child {
 			margin-bottom: 0;
 		}
 
 		code {
-			padding: 0;
+			white-space: pre;
```

Comment:
Can you reorganize this code with display styles first, with the rest listed alphabetically?